_posts/2025-03-10-mpact_cheriot.markdown

```diff
@@ -0,0 +1,470 @@
+# MPACT-Cheriot
```

Comment:
Can you add YAML front matter for the title and author?

Reply:
We don’t need the title and author here, it will be generated from the front matter. Currently this will show the title and author twice.